changelog/changes/serve-comic-ending.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Fix stack traces in the docker image build"
```

Comment:
```suggestion
title: "Fixed stack traces in Docker images"
```

Reply:
The template created by the changelog bot should really not give any default for the title. The PR title is pretty much always the wrong style.